Sprintr Release 2023.07.27 (#6345)
mendix/docs: content/en/docs/releasenotes/developer-portal/_index.md

## 2023

### July 27, 2023

#### Improvements

* In Mendix 10, we have combined the strength of Mendix Studio and Studio Pro - they are not separate anymore, so we have removed the last links from the Developer Portal to Studio.

> * We have improved the information regarding the sunset of [Mendix Stories](/developerportal/project-management/stories/) (remember, it's October 1)

Reviewer: @loket4 
Hi Olaf - should we put 2024 here as well, so as not to scare the customers?

Author: The functionality will be removed on 1 October 2023. The data will be available to customers until 1 October 2024. In thi case I would mention 1 October 2023.

Reviewer: OK, let's add 2023 to this note just to be absolutely clear!
```suggestion
* We have improved the information regarding the sunset of [Mendix Stories](/developerportal/project-management/stories/) (remember, it's October 1, 2023)
```

> * We are preparing the platform for the [big upgrade of the feedback management experience](https://www.mendix.com/blog/mendix-release-10-0-a-new-era/#app-insights). The new feedback management and mini surveys are a part of our brand new product suite called App Insights. The date for this upgrade of feedback management is set to August 12. If you are using the current feedback management tool, you also received more detailed information in an email sent on Monday, July 24 (subject line: Upgrade to new Feedback Management).

Reviewer: @loket4 
Lakshya just told me August 13th.
```suggestion
* We are preparing the platform for the [big upgrade of the feedback management experience](https://www.mendix.com/blog/mendix-release-10-0-a-new-era/#app-insights). The new feedback management and mini surveys are a part of our brand new product suite called App Insights. The date for this upgrade of feedback management is set to August 13. If you are using the current feedback management tool, you will have received more detailed information in an email sent on Monday, July 24 (subject line: Upgrade to new Feedback Management).
```

> 

Author: Please add the following line:
You asked, we listened: In [Epics](https://docs.mendix.com/developerportal/project-management/epics/), you can now see who created a story on the story details page.

Reviewer: ```suggestion
* You asked, we listened: In Epics, you can now see who created a story on the story details page.

```
